### ISSUE TITLE
svg text rendered improperly (way too big)

### ISSUE THREAD

**Issue by timeisapear** [Kozea/CairoSVG#71]
When rendering a simple axis label with numerical values, any values enclosed in `<text></text>` tags are rendered improperly.

Stripped down version of D3 chart, after applying inline styles (needed in weasyprint svg pipeline).

Renders (incorrectly) using CairoSVG as:
<img width="621" alt="screen shot 2015-07-21 at 5 13 52 pm" src="https://cloud.githubusercontent.com/assets/4983212/8812742/f8c41816-2fcb-11e5-8ba7-6463c068af23.png">

Renders (correctly) in the browser as: 
![screen shot 2015-07-21 at 5 12 14 pm](https://cloud.githubusercontent.com/assets/4983212/8812712/c4b1ebfc-2fcb-11e5-8bb8-e4b632cf47e2.png)

```html
<svg width="600" height="300">
    <g transform="translate(15,6)">
        <g class="xaxis" transform="translate(0,70)">
            <g class="tick major" style="opacity: 1;" transform="translate(0,0)">
                <line y2="-10" x2="0" fill="none" stroke="#999999" shape-rendering="crispEdges"></line>
            </g>
            <g class="tick major" style="opacity: 1;" transform="translate(60.64680746308627,0)">
                <line y2="-10" x2="0" fill="none" stroke="#999999" shape-rendering="crispEdges"></line>
            </g>
            <g class="tick major" style="opacity: 1;" transform="translate(121.29361492617254,0)">
                <line y2="-10" x2="0" fill="none" stroke="#999999" shape-rendering="crispEdges"></line>
            </g>
            <g class="tick major" style="opacity: 1;" transform="translate(179.41347207829688,0)">
                <line y2="-10" x2="0" fill="none" stroke="#999999" shape-rendering="crispEdges"></line>
            </g>
            <g class="tick major" style="opacity: 1;" transform="translate(240.06027954138315,0)">
                <line y2="-10" x2="0" fill="none" stroke="#999999" shape-rendering="crispEdges"></line>
            </g>
            <g class="tick major" style="opacity: 1;" transform="translate(300.70708700446943,0)">
                <line y2="-10" x2="0" fill="none" stroke="#999999" shape-rendering="crispEdges"></line>
            </g>
            <g class="tick major" style="opacity: 1;" transform="translate(361.3538944675557,0)">
                <line y2="-10" x2="0" fill="none" stroke="#999999" shape-rendering="crispEdges"></line>
                <rect width="60.64680746308627" y="-70" height="70" class="bar-today" fill="#999999" opacity="0.4" shape-rendering="crispEdges"></rect>
            </g>
            <path class="domain" d="M0,-10V0H422V-10" fill="none" stroke="#999999" shape-rendering="crispEdges"></path>
        </g>
        <g transform="translate(422,0)">
            <g class="tick major" transform="translate(0,70)">
                <text style="font-size:12px; font-family:'DIN Light', sans-serif;">420</text>
            </g>
            <g transform="translate(0,0)">
                <text style="font-size:12px; font-family:'DIN Light', sans-serif; ">600</text>
            </g>
            <path class="domain" d="M6,0H0V70H6" fill="none" stroke="#999999" shape-rendering="crispEdges"></path>
        </g>
    </g>
</svg>
```

**Comment by liZe** [comment]
Thanks for reporting.

I can't reproduce the bug, the image is rendered correctly for me. This bug looks like an odd bug in Pango or Cairo we had seen with WeasyPrint on OSX.

What OS are you using, and if you're on OSX could you try on another OS please? I'll try to find the bug that was reported on WeasyPrint and see if there's something in it that can help you to solve the problem.

**Comment by timeisapear** [comment]
Yes, I attempted this on Mac OS X.

Because our project eventually requires the use of Windows, I have attempted to install weasyprint/cairosvg on there but have failed so far.  For portability reasons, we utilize [winpython](http://winpython.github.io) as the install base on Windows as opposed to CPython. I tried installing weasyprint and the dependencies (gtk, etc.) as described [here](http://weasyprint.org/docs/install/) but I am getting an error like `OSError: cannot load library 'libcairo.so.2': error 0x7e`. There are no .so files in the installation instructions linked GTK+ installer for windows, only .dlls. Winpythons `%PATH%` variable includes the `gtk/bin` directory.

Any ideas on the install at least?
